packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -580,15 +580,15 @@ Fetch a list of ordered log events from your Okta organization's system log.
-| `sortOrder` | The order of the returned events sorted by `published`                                                                                  | `ASCENDING` or `DESCENDING`                                                                                                         | `ASCENDING`             |
-| `limit`     | Sets the number of results returned in the response                                                                                     | Integer between 0 and 1000                                                                                                          | 100                     |
+| Parameter   | Description                                                                                                                             | Format                                                                                                                                                            | Default                 |
+| ----------- | -----------------------------------------------------------------------------------------------------                                   | --------------------------------------------------------                                                                                                          | ----------------------- |
+| `since`     | Filters the lower time bound of the log events `published` property                                                                     | The [Internet Date/Time Format profile of ISO 8601](https://tools.ietf.org/html/rfc3339#page-8). An example: `2017-05-03T16:22:18Z`                               | 7 days prior to `until` |
+| `until`     | Filters the upper time bound of the log events `published` property                                                                     | The [Internet Date/Time Format profile of ISO 8601](https://tools.ietf.org/html/rfc3339#page-8). An example: `2017-05-03T16:22:18Z`                               | Current time            |
+| `after`     | Used to retrieve the next page of results. Okta returns a link in the HTTP Header (`rel=next`) that includes the after query parameter. | Opaque token                                                                                                                                                      |                         |
+| `filter`    | [Filter Expression](#expression-filter) that filters the results                                                                        | [SCIM Filter expression](/docs/reference/api-overview/#filtering). All [operators](https://tools.ietf.org/html/rfc7644#section-3.4.2.2) except `ne` are supported |                         |
```

Comment:
I'm try to square this comment with [OKTA-188737](https://oktainc.atlassian.net/browse/OKTA-188737):

![image](https://user-images.githubusercontent.com/26014539/59947420-12fed400-9422-11e9-8d74-59fb831b32f6.png)

Both can't be correct, right?

Reply:
Good catch! Just noticed the table went on to the next page!

Will fix.

Reply:
Btw, pr works today